client/src/pages/Admin/UserDetails.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <article>
+    <nav class="q-px-lg q-pt-md q-gutter-sm">
+      <q-breadcrumbs>
+        <q-breadcrumbs-el
+          label="All Users"
+          to="/admin/users"
+        />
+        <q-breadcrumbs-el label="User Details" />
+      </q-breadcrumbs>
+    </nav>
+    <div class="row justify-center q-px-lg">
+      <h2
+        class="col-sm-12"
+        data-cy="userDetailsHeading"
+      >
+        {{ user.username || `Loading...` }}
```

Comment:
This would be better handled with a containing element with v-if that checked the graphql loading status. 

https://apollo.vuejs.org/guide/apollo/queries.html#loading-state

Reply:
Resolved